Gordon360/Models/CCT/Clifton_Strengths.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Gordon360.Models.CCT
 {
+    [Table("Clifton_Strengths", Schema = "dbo")]
```

Comment:
Because we have 2 schema's within the CCT database, EF Core now needs to recognize the difference between the default dbo schema and our RecIM schema. There are no other examples within 360 which is why it may look new.